aurelia-slickgrid/src/examples/slickgrid/example3.ts

```diff
@@ -308,7 +310,9 @@ export class Example3 {
         dataKey: 'code',
         labelKey: 'name',
         type: FieldType.object,
+        sorter: Sorters.objectString,
```

Comment:
change it to
```ts
sorter: Sorters.objectString, // this sorter requires the dataKey and assume that obj1[dataKey] will be a string so it can sort it that way
```